src/editor/icons/index.js

```diff
@@ -32,5 +32,7 @@ export {
   SignInMicrosoftIconSVG,
   Rotate24Icon,
   Translate24Icon,
-  Object24Icon
+  Object24Icon,
+  SegmentIcon,
+  ManagedStreetIcon
```

Comment:
Please can we replace this file by `export * from './icons.jsx';` and be done with those changes.

Reply:
bdcc04f89b35d76d0349b4da0ca4c4b52365ba7f